cftime/_cftime.pyx

```diff
@@ -1555,8 +1560,15 @@ Gregorial calendar.
         return self.__class__(**args)
 
     def timetuple(self):
-        return (self.year, self.month, self.day, self.hour,
-                self.minute, self.second, self.dayofwk, self.dayofyr, -1)
+        """
+        Return a time.struct_time such as returned by time.localtime().
+        The hours, minutes and seconds are 0, and the DST flag is -1.
+        d.timetuple() is equivalent to time.struct_time((d.year, d.month, d.day,
+        0, 0, 0, d.weekday(), yday, -1)), where yday is the day number within
```

Comment:
Here too

Reply:
Corrected. I kept "The DST flag is -1" since that's true for the cftime implementation.